<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oui</class>
 <widget class="QWidget" name="oui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>oui</string>
  </property>
  <widget class="QPushButton" name="pushButton_Quitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OuvrirPortSerie">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ouvrir le Port Série</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Envoyer">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Port">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Choisissez un Port Série</string>
   </property>
   <item>
    <property name="text">
     <string>Choisissez un Port Série</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Texte à Envoyer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Texte Reçu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TexteEnvoyer">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>70</y>
     <width>411</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_TexteRecu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>611</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>oui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
